doc/examples/reconst_mapmri.py

```diff
@@ -143,136 +146,198 @@
 and also faster in practice [Fick2016a]_.
 
-We can then fit the MAPMRI model to the data:
+We can then fit the MAP-MRI model to the data:
 """
 
 mapfit_laplacian_aniso = map_model_laplacian_aniso.fit(data_small)
 mapfit_positivity_aniso = map_model_positivity_aniso.fit(data_small)
 mapfit_both_aniso = map_model_both_aniso.fit(data_small)
+mapfit_plus_aniso = map_model_plus_aniso.fit(data_small)
 
 """
 From the fitted models we will first illustrate the estimation of q-space
-indices. For completeness, we will compare the estimation using only Laplacian
-regularization, positivity constraint or both. We first show the RTOP
-[Ozarslan2013]_.
+indices. We will compare the estimation using only Laplacian regularization,
+using only the global positivity constraint, using only positivity in discrete
+points, or using both Laplacian regularization and positivity in discrete
+points. We first show the RTOP [Ozarslan2013]_.
 """
 
-# generating RTOP plots
-fig = plt.figure(figsize=(10, 5))
-ax1 = fig.add_subplot(1, 3, 1, title=r'RTOP - Laplacian')
+fig = plt.figure(figsize=(18, 3))
+
+ax1 = fig.add_subplot(1, 5, 1, title=r'RTOP - Laplacian')
 ax1.set_axis_off()
-rtop_laplacian = np.array(mapfit_laplacian_aniso.rtop()[:, 0, :].T,
-                          dtype=float)
-ind = ax1.imshow(rtop_laplacian, interpolation='nearest',
-                 origin='lower', cmap=plt.cm.gray)
+rtop_laplacian = np.array(mapfit_laplacian_aniso.rtop()[:, 0, :].T, dtype=float)
+ind = ax1.imshow(rtop_laplacian, interpolation='nearest', origin='lower',
+                 cmap=plt.cm.gray)
 
-ax2 = fig.add_subplot(1, 3, 2, title=r'RTOP - Positivity')
+ax2 = fig.add_subplot(1, 5, 2, title=r'RTOP - Discrete positivity')
 ax2.set_axis_off()
 rtop_positivity = np.array(mapfit_positivity_aniso.rtop()[:, 0, :].T,
                            dtype=float)
-ind = ax2.imshow(rtop_positivity, interpolation='nearest',
-                 origin='lower', cmap=plt.cm.gray)
+ind = ax2.imshow(rtop_positivity, interpolation='nearest', origin='lower',
+                 cmap=plt.cm.gray)
 
-ax3 = fig.add_subplot(1, 3, 3, title=r'RTOP - Both')
+ax3 = fig.add_subplot(1, 5, 3, title=r'RTOP - Both')
 ax3.set_axis_off()
 rtop_both = np.array(mapfit_both_aniso.rtop()[:, 0, :].T, dtype=float)
 ind = ax3.imshow(rtop_both, interpolation='nearest', origin='lower',
                  cmap=plt.cm.gray)
-divider = make_axes_locatable(ax3)
-cax = divider.append_axes("right", size="5%", pad=0.05)
+
+ax4 = fig.add_subplot(1, 5, 4, title=r'RTOP - Global positivity')
+ax4.set_axis_off()
+rtop_plus = np.array(mapfit_plus_aniso.rtop()[:, 0, :].T, dtype=float)
+ind = ax4.imshow(rtop_plus, interpolation='nearest', origin='lower',
+                 cmap=plt.cm.gray)
+
+ax5 = fig.add_subplot(1, 5, 5)
+ax5.set_axis_off()
+divider = make_axes_locatable(ax5)
+cax = divider.append_axes("left", size="10%")
 plt.colorbar(ind, cax=cax)
 
-plt.savefig('MAPMRI_maps_regularization.png')
+plt.savefig('MAPMRI_rtop.png')
```

Comment:
Hi @tomdelahaije,

Like we talked about last time, can you move all `Matplotlib` functions in a new module inside `dipy.viz`?

Thank you